transform/mattermost-analytics/models/marts/sales/hightouch/_hightouch__models.yml

```diff
@@ -62,3 +62,26 @@ models:
         description: The campaign member's status.
       - name: campaign_id
         description: The id of the campaign to attribute this lead to.
+  - name: fct_cloud_trial_requests
+    description: |
+      Trial request data coming from the source Stripe, only limited to creating campaign member objects in Salesforce.
+
+    columns:
+      - name: email
+        tags: ['pii']
+        description: Customer email for cloud trial requests coming from Stripe.
+        tests:
+          - not_null
+          - dbt_utils.not_empty_string
+      - name: is_existing_lead
+        description: Whether there's already a lead with the same email.
+      - name: is_existing_campaign_member
+        description: Whether there's already a campaign member for the same email and lead.
+      - name: campaign_member_status
+        description: The campaign member's status.
+        # Removing the test for now.
```

Comment:
Why is this test removed?

Reply:
Campaign member status is set to `Workspace verified` for all the leads, for other status we need to update that as part of another ticket.